guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-hosts.adoc

```diff
@@ -10,6 +10,5 @@ After you configure {ProductName} to use a custom SSL certificate, you must also
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# yum localinstall \
-http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
+# {package-install} http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
```

Comment:
```suggestion
# {package-install} http://_{project-context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
```

IMHO this is a suitable alternative because a) lowercase and b) no whitespace. See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L82

Reply:
Can you submit that as a separate PR? It'll make the diff easier to read.